README.md

```diff
@@ -1 +1,43 @@
-# capstone-alpha-one
\ No newline at end of file
+# capstone-alpha-one
+
+This is my capstone project for module one.
+
+## Built With
+- HTML
+- CSS
+- Javascript
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@godiarksel](https://github.com/githubhandle)
+- Twitter: [@godiarksel](https://twitter.com/twitterhandle)
+- LinkedIn: [Godfrey Mwandembo](https://linkedin.com/in/godfrey-mwandembo-045667127/)
+
+
+##  Attributions
+
+   1. [Lecture icons created by berkahicon - Flaticon](https://www.flaticon.com/free-icons/lecture)
+   2. [Tour icons created by Smashicons - Flaticon](https://www.flaticon.com/free-icons/tour)
+   3. [Flaticon Author: HAJICON](https://www.flaticon.com/authors/hajicon)
+   4. [Flaticon Author: gungyoga04](https://www.flaticon.com/authors/gungyoga04)
+   5. [Freepik.com](https://www.freepik.com)
+   6. [Flaticon Author: Pixel perfect](https://www.flaticon.com/authors/pixel-perfect)
+   7. [Pixabay Author: Gordon Johnson](https://pixabay.com/users/gdj-1086657/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=5650230)
+   8. [Pixabay Author: Vizetelly](https://pixabay.com/users/vizetelly-6208731/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amputm_content=6363862)
+   9. [White Design Vector created by freepik.com](https://www.freepik.com/vectors/white-design)
+   10. [Dark Blue Background Vectors by Vecteezy](https://www.vecteezy.com/free-vector/dark-blue-background)
+
+
+
+## Acknowledgments
+
+- The Great Ancient Philosophers who inspired this project. 
+- Fellow Micronauts
+- Myself
+
+## Live Preview
+
+https://godiarksel.github.io/capstone-alpha-one/
```

Comment:
- [OPTIONANL] Please update your Readme, currently some headings are missing such as  

``` ##  Getting Started ```

It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc

- I would also recommend to add the following: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```


- please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.